.github/workflows/docker-tests.yml

```diff
@@ -4,12 +4,15 @@ on:
     paths-ignore:
       - '**/*.md'
   pull_request_target:
```

Comment:
remove pull_request_target here, and branch cond'n as well, just make it run on push event